modules/sections.js

```diff
@@ -0,0 +1,41 @@
+const section = () => {
+  document.addEventListener('DOMContentLoaded', () => {
+    const $first = document.querySelector('.first');
+    const $showLib = document.querySelector('.showLib');
+    const $titulo = document.querySelector('.titleawe');
+    const $Addnew = document.querySelector('.addnew');
+    const $plus = document.querySelector('.plus');
+    const $cont = document.getElementById('Button');
+    const $Information = document.querySelector('.Inf');
+    const $foot = document.querySelector('.myfoot');
```

Comment:
- [ ] [OPTIONAL] Do not start names with a $  sign since it will put you in conflict with many JavaScript library names. Also, you should try as much as possible to use the camelCase naming convention in order to conform with the JavaScript standard naming convention.  Kindly check this [link](https://www.w3schools.com/js/js_conventions.asp) for more details on that.